labs/ipc/skel/win/utils/utils.h

```diff
@@ -1,20 +1,19 @@
 /**
- * SO, 2011
+ * SO
  * Windows
  *
  * Useful structures/macros
  */
-
 #ifndef WIN_UTILS_H_
 #define WIN_UTILS_H_	1
 
-#undef UNICODE
+#undef UNICODE
 #undef _UNICODE
 
 #include <windows.h>
 #include <stdio.h>
 
-/* useful macro for handling error codes */
+/* Useful macro for handling error codes */
 #define DIE(assertion, call_description)		\

```

Comment:
Replace this `DIE` with
```c
#define DIE(assertion, call_description)			\
	do {							\
		if (assertion) {				\
			fprintf(stderr, "(%s, %s, %d): ",	\
			__FILE__, __FUNCTION__, __LINE__);	\
			PrintLastError(call_description);	\
			exit(EXIT_FAILURE);			\
		}						\
	} while(0)
```

So that the `\`s are aligned.